data/remnant missions.txt

```diff
@@ -495,3 +560,759 @@ mission "Remnant: Technology Available"
+			`	It occurs to you that Dawn and her colleagues seem much less reserved than most Remnant. You aren't sure if it is because they are younger, less secretive, or if the Remnant have been concealing it from you. It is also possible that you just have better opportunities to figure it out now that you can understand their signs.`
+			`	Upon arriving at Wayfarer they spend considerable amount of time watching the few Quarg in evidence, however it is easy to see that the real draw of their attention is the ships. You wave your ID across the scanner for the shipyard and take them inside to see what Tarazed Shipyards has on display.`
+			branch rich
+				"net worth" >= 20000000
+			`	After a few minutes a sales agent meanders over to ask if there's anything he can do to help. You ask a few questions about the different ships for Dawn's benefit, listen to the sales agent repeat a few lines from the display, then politely tell him you'll just browse on your own.`
+				goto end
+			label rich
+			`	Almost before you are in the main hall a sales manager is at your side, inquiring as to how he can help. You can almost see the credit signs in his eyes and you suspect that scanning your license may have done a quick check of your net worth too.`
+			`	You ask a few leading questions about new designs and play the part of a captain with more money than brains, and the manager eagerly directs you to the large hangars near the end that house their largest and fanciest ships. After it becomes clear that he doesn't have any experience flying them you politely disentangle yourselves from him to look at the ships in peace.`
+			label end
+			`	After an hour of browsing, the young remnant take off to browse the outfitters and agree to meet you back at the spaceport in an hour.`
+
+mission "Remnant: Expanded Horizons Quarg 2"
+	name "Quarg Scanning"
+	description "Perform an outfit scan on the Quarg ship <npc>, then land again on <destination>."
+	source
+		planet Wayfarer
+	to offer
+		has "Remnant: Expanded Horizons Quarg 1: done"	
+	on offer
+		conversation
+			`Dawn and her fellow researchers show up a few minutes after you do, excitedly talking about the various things they have seen.`
+			`	"<first>, there is a pair of Quarg ships taking off shortly, and we were hoping you might be able to get scans of them while they are in flight. It would be really helpful for our understanding of how they work."`
+			choice
+				`	"Sure, that sounds simple enough."`
+					accept
+				`	"Sorry, that sounds risky."`
+					defer
+
+	on accept
+		log "Volunteered to show Dawn and some of her fellow researchers the world where humans and Quarg co-exist."
+	
+	npc "scan outfits"
+		government Quarg
+		personality staying uninterested
+		fleet Quarg
+		conversation
+			`	Dawn and her compatriots eagerly pore over the readouts as they come in, then ask if you could land again to compare their scans to ones taken on the ground.`
```

Comment:
This is short and could therefore be a dialog box instead of a conversation block. And even if it was long enough to warrant being a conversation, the first line wouldn't need tabbed in like this.